packs/data/extinction-curse-bestiary.db/shraen-graveknight.json

```
@@ -1392,10 +1570,81 @@
             "type": "action"
         },
         {
-            "_id": "dH51ukXnw1VQSb6t",
+            "_id": "BeWl7Ot39HAvDQoK",
+            "img": "systems/pf2e/icons/actions/Passive.webp",
+            "name": "Status Bonus to Saves",
+            "sort": 2100000,
+            "system": {
+                "actionCategory": {
+                    "value": "defensive"
+                },
+                "actionType": {
+                    "value": "passive"
+                },
+                "actions": {
+                    "value": null
+                },
+                "description": {
+                    "value": "<p>+1 status bonus to all saves vs. magic effects.</p>\n<p>+2 status bonus to all saves vs. mental effects.</p>"
+                },
+                "requirements": {
+                    "value": ""
+                },
+                "rules": [
+                    {
+                        "key": "FlatModifier",
+                        "label": "Vs. Magic Effects",
```

```suggestion
                        "label": "PF2E.SkillVariant.VsMagic",
                        "slug": "status-vs-magic",
```
I'll add these keys later, want to get these PRs cleared out before migrations start back up for the system.